tfhe/src/shortint/engine/client_side.rs

```diff
@@ -65,16 +66,11 @@ impl ShortintEngine {
         u64: RandomGenerable<NoiseDistribution, CustomModulus = u64>,
         KeyCont: crate::core_crypto::commons::traits::Container<Element = u64>,
     {
-        //The delta is the one defined by the parameters
-        let delta = (1_u64 << 63)
-            / (client_key_parameters.message_modulus().0 * client_key_parameters.carry_modulus().0);
-
-        //The input is reduced modulus the message_modulus
+        // The input is reduced modulus the message_modulus
```

Comment:
This comment is not really useful and could be removed IMO
Applies to many places